charts/posthog/templates/ingress.yaml

```diff
@@ -56,8 +56,13 @@ spec:
     - http:
   {{- end }}
         paths:
+          {{- if (ne (include "ingress.type" .) "clb") }}
```

Comment:
nit: 
instead of 
```
if not x:
  y 
else:
  z
```
I prefer
```
if x:
 z 
else:
  y
```

Reply:
PR accepted ;)

Reply:
yup totally didn't mean for you to update again, that was more for future reference ;)